apps/events.md

```diff
@@ -5,21 +5,24 @@ title: Events
 ---
 
 Events can be triggered when a customer executes a specific action in an ePages online shop.
-This can be e.g. a customer opening a new page of a shop, or adding a product to the cart.
+This can, for example, be a customer opening a new page of a shop, or adding a product to the cart.
 If you have implemented an event in your application, you will be informed about related actions and can react accordingly.
 Furthermore, your app will receive information related to the event, such as the current page a user is on.
 
 These events are available:
 
-| Event | Type | Description |
-| - | - |  - |
-| `pageview` | string | Is triggered when the customer opens or reloads a page. Informs about the path of the page. |
-| `product` | immutable.js object | Is triggered when the customer selects a product. Informs about the respective product as well as the current state of the cart. |
-| `category` | immutable.js object | Is triggered when the customer selects a category. Informs about the respective category and related products. |
-| `cart` | object | Is triggered when the customer accesses the cart. Informs about the current state of the cart, e.g. included items. |
-| `cart:add` | object | Is triggered when the customer adds a product to the cart. Provides information about the current state of the cart and the product that was added to it. |
-| `cart:setQuantity` | object | Is triggered when the quantity of a product in the cart is changed. This also includes the removal of a product. Informs about the quantity change and the affected product. |
-| `order:completed` | object | Is triggered when the customer reaches the order confirmation page after completing a purchase. Provides additional information about the order, such as the billing address, the order number, and the selected shipping method. |
+| Event | Description |
```

Comment:
I've decided to remove the type column as I do not think that it has much value in this condensed version. The long version like on https://github.com/ePages-de/epages-ui/commit/cfa86e498 would be way to long. As developers can see the type in the example below, they can still get that information :)

Reply:
Good point. I'm not sure if each developer has access to https://github.com/ePages-de/epages-ui 🤔

Reply:
@bulutfatih For sure, they do not have access. But my point is that we have two options:

A: Extend the type information and copy the information that is stated in epages-ui in order to have a meaningful content for that column.

B: Remove the type column as we have the "You'll receive the following information:" part for each event. Developers can thus see the type themselves if they are interested in it (without the need to access epages-ui)

As I didn't want to have the big type column, I went with option B :)